docs/install.md

```diff
@@ -93,3 +93,32 @@ To update the News app use change into the **nextcloud/apps/news/** directory us
 
     git pull --rebase origin master
     make
+
+## Uninstall with cleanup
+
+First uninstall the app via the web-interface or via occ:
+    
+    ./occ app:remove news
+
+This currently does not remove any of the database tables.
+Data in your **/tmp** directory will be automatically deleted by the OS.
+If you changed the temporary directory for Nextcloud you need to check on your own.
+
+To remove the tables from the DB we drop the tables of news.
+Your installation might have a different prefix then **oc_** but it is the default in most installations.
+Connect to your DB and execute the commands. Don't forget to switch to the right DB within your DB.
+For example in mysql: **use nextcloud;**
+
+    DROP TABLE oc_news_folders;
+    DROP TABLE oc_news_feeds;
+    DROP TABLE oc_news_items;
+
+Then we remove the traces in the migrations table.
+
+    DELETE FROM oc_migrations WHERE app='news';
+
+And finally remove the jobs from the job table.
+
+    DELETE FROM oc_jobs WHERE class LIKE '%News%';
+
```

Comment:
I'm not sure we should be doing this

Reply:
hmm yea maybe News is too broad.
I will add two lines instead to delete the old and new job with exact naming.

Reply:
I'm not sure how old this job entry is, but I've also found this in my db:
| class | argument |
|:--|:--|
| `OC\BackgroundJob\Legacy\RegularJob` |  `["OCA\\News\\Cron\\Updater","run"]` |